system-tests/scripts/bootstrap-docker-container.sh

```diff
@@ -36,6 +36,9 @@ export CYPRESS_CACHE_FOLDER=/tmp/CYPRESS_CACHE_FOLDER/
 export npm_config_cache=/tmp/npm_config_cache/
 export npm_config_package_lock=false
 
+mkdir /tmp/npm_config_cache
+chown -R 1000:1000 /tmp/npm_config_cache
```

Comment:
Since it's already a variable:

```suggestion
mkdir $npm_config_cache
chown -R 1000:1000 $npm_config_cache
```

Reply:
I guess we do need the `chown`: https://app.circleci.com/pipelines/github/cypress-io/cypress/42916/workflows/07a14aa6-9dea-4224-861e-7776e632f27f/jobs/1787330

Reply:
I reverted and now CI is green again. 💯